Tests/ComposableArchitectureTests/StoreTests.swift

```diff
@@ -931,6 +931,84 @@ final class StoreTests: BaseTCATestCase {
     store.send(.child(.dismiss))
     _ = (childViewStore1, childViewStore2, childStore1, childStore2)
   }
+
+  func testStoreDeinit() {
+    Logger.shared.isEnabled = true
+    defer { Logger.shared.isEnabled = false }
+    Logger.shared.clear()
```

Comment:
down to move this into the base test case like you were about to do, or move these 3 tests to their own test file.